### DISCUSSION TITLE
Create electron rclnodejs project

### DISCUSSION THREAD

**Discussion by AceOfDema** [RobotWebTools/rclnodejs#994] [Q&A] [answered]
Hi my first project was using express for the web part now i wish to remake that but with electron does someone know how i should approach it? first create the package like usual and delete src include and create insiode the electron app? its my first time so im asking for some guidace

**Comment by minggangw** [comment]
I think using `rclnodejs` with Electron should be the same as other modules, if you meet some issues you can search https://github.com/RobotWebTools/rclnodejs/issues?q=electron+

**Reply by minggangw** [reply, marked as answer]
@AceOfDema I have added a Electron demo for rclnodejs https://github.com/RobotWebTools/rclnodejs/tree/develop/electron_demo, please try it.
